authmodules/TYPO3/privacyidea/Classes/PrivacyideaService.php

```diff
@@ -110,23 +113,30 @@ public function init() {
 	 * @return int authentication statuscode, one of 0, 100 and 200
 	 */
 	public function authUser(array $user) {
-		// 0 means authentication failure
-		$ret = 0;
-		$username = $this->login['uname'];
-		$password = $this->login['uident_text'];
-		$this->logger->info("try to authenticate user [$username]");
+		if ($this->extConf['privacyIDEABackend'] != 'adminOnly'
+			|| (bool)$user['admin']
+			|| $this->authInfo['loginType'] == 'FE'
+		) {
```

Comment:
I would like to have the adminOnly functionality because:
- if the auth service is broken, normal users can still login
- the audit would otherwise contain personal data (IP addresses) of our customers
- I can distinguish between password logins and failed 2factor auths at first sight

"allUsers" is a better term, changed it.